.github/workflows/ci.yml

```
@@ -0,0 +1,55 @@
+name: CI
```

```suggestion
name: Build PR
```

Also rename the file to `build-pr.yml`.

In the next PR, we should create `build-develop.yml` to build and upload to beta when the PR gets merged to develop branch.